components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConstants.java

```diff
@@ -400,6 +400,12 @@
     //Property for enabling scope sharing between APIs
     public static final String ENABLE_API_SCOPES_SHARING = "enable-api-scopes-sharing";
 
+    // Constants for obtaining organization claims
+    public static final String CHECK_ORGANIZATION_FROM_SAML_ASSERTION = "checkOrganizationFromSamlAssertion";
```

Comment:
shall we rename this to "readOrganizationClaimFromSamlAssertion"? (note read and claim parts.)